### PR TITLE
[Doc] 3.2 MV doc updates

### PR DESCRIPTION
**Pull request by EsoragotoSpirit**
Signed-off-by: EsoragotoSpirit <wanglichen@starrocks.com>

Fixes #issue

## What type of PR is this:

- [ ] BugFix
- [ ] Feature
- [ ] Enhancement
- [ ] Refactor
- [ ] UT
- [x] Doc
- [ ] Tool

Does this PR entail a change in behavior?

- [ ] Yes, this PR will result in a change in behavior.
- [x] No, this PR will not result in a change in behavior.

If yes, please specify the type of change:

- [ ] Interface/UI changes: syntax, type conversion, expression evaluation, display information
- [ ] Parameter changes: default values, similar parameters but with different default values
- [ ] Policy changes: use new policy to replace old one, functionality automatically enabled
- [ ] Feature removed
- [ ] Miscellaneous: upgrade & downgrade compatibility, etc.

## Checklist:

- [ ] I have added test cases for my bug fix or my new feature
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function

## Bugfix cherry-pick branch check:

- [x] I have checked the version labels which the pr will be auto-backported to the target branch
  - [x] 3.2
  - [ ] 3.1
  - [ ] 3.0
  - [ ] 2.5

### CODE REVIEW

**Comment by wanpengfei-git** [comment]
#### [FE Incremental Coverage Report]


:heart_eyes: **pass** : 0 / 0 (0%)

**Comment by wanpengfei-git** [comment]
#### [BE Incremental Coverage Report]


:heart_eyes: **pass** : 0 / 0 (0%)

**Comment by wanpengfei-git** [comment]
#### [FE Incremental Coverage Report]


:heart_eyes: **pass** : 0 / 0 (0%)

**Comment by wanpengfei-git** [comment]
#### [BE Incremental Coverage Report]


:heart_eyes: **pass** : 0 / 0 (0%)

File: docs/sql-reference/sql-statements/data-definition/CREATE MATERIALIZED VIEW.md [hunk 1]
```diff
@@ -255,6 +255,17 @@ Properties of the asynchronous materialized view. You can modify the properties
 - `colocate_with`: The colocation group of the asynchronous materialized view. See [Colocate Join](../../../using_starrocks/Colocate_join.md) for further information. This property is supported from v3.0.
 - `unique_constraints` and `foreign_key_constraints`: The Unique Key constraints and Foreign Key constraints when you create an asynchronous materialized view for query rewrite in the View Delta Join scenario. See [Asynchronous materialized view - Rewrite queries in View Delta Join scenario](../../../using_starrocks/Materialized_view.md#rewrite-queries-in-view-delta-join-scenario) for further information. This property is supported from v3.0.
 - `resource_group`: The resource group to which the refresh tasks of the materialized view belong. For more about resource groups see [Resource group](../../../administration/resource_group.md).
+- `query_rewrite_consistency`: The query rewrite rule the materialized view. This property is supported from v3.2. Valid values:
```
Comment (evelynzhaojie) [inline docs/sql-reference/sql-statements/data-definition/CREATE MATERIALIZED VIEW.md:258]: incomplete sentence, maybe "for" is missing: The query rewrite rule for asynchronous materialized views.

File: docs/sql-reference/sql-statements/data-definition/CREATE MATERIALIZED VIEW.md [hunk 2]
```diff
@@ -255,6 +255,17 @@ Properties of the asynchronous materialized view. You can modify the properties
...
+  - `disable`: Disable the automatic query rewrite using the asynchronous materialized view.
```
Comment (evelynzhaojie) [inline docs/sql-reference/sql-statements/data-definition/CREATE MATERIALIZED VIEW.md:259]: Disable the automatic query rewrite --> Disable automatic query rewrite

File: docs/sql-reference/sql-statements/data-definition/CREATE MATERIALIZED VIEW.md [hunk 3]
```diff
@@ -255,6 +255,17 @@ Properties of the asynchronous materialized view. You can modify the properties
...
+  - `checked` (Default value): Enable the automatic query rewrite only when the materialized view meets the timeliness requirement, which means:
```
Comment (evelynzhaojie) [inline docs/sql-reference/sql-statements/data-definition/CREATE MATERIALIZED VIEW.md:260]: Enable the automatic query rewrite --> Enable automatic query rewrite

File: docs/sql-reference/sql-statements/data-definition/CREATE MATERIALIZED VIEW.md [hunk 4]
```diff
@@ -255,6 +255,17 @@ Properties of the asynchronous materialized view. You can modify the properties
...
+    - if `mv_rewrite_staleness_second` is not specified, the materialized view can be used for query rewrite only when its data is consistent with the data in all base tables
```
Comment (evelynzhaojie) [inline docs/sql-reference/sql-statements/data-definition/CREATE MATERIALIZED VIEW.md:261]: capitalize "If", same for line 262

File: docs/reference/System_variable.md [hunk 5]
```diff
@@ -357,6 +357,16 @@ Specifies the maximum number of unqualified data rows that can be logged. Valid
 
 Used for MySQL client compatibility. No practical usage. Table names in StarRocks are case-sensitive.
 
+### materialized_view_rewrite_mode (v3.2 and later)
+
+Specifies the query rewrite mode of asynchronous materialized views. Valid values:
+
+* `disable`: Disable the automatic query rewrite of asynchronous materialized views.
```
Comment (evelynzhaojie) [inline docs/reference/System_variable.md:364]: Disable automatic query rewrite of xxx, same for the following items

**Comment by wanpengfei-git** [comment]
#### [FE Incremental Coverage Report]


:heart_eyes: **pass** : 0 / 0 (0%)

**Comment by wanpengfei-git** [comment]
#### [BE Incremental Coverage Report]


:heart_eyes: **pass** : 0 / 0 (0%)

**Comment by wanpengfei-git** [comment]
#### [FE Incremental Coverage Report]


:heart_eyes: **pass** : 0 / 0 (0%)

**Comment by wanpengfei-git** [comment]
#### [BE Incremental Coverage Report]


:heart_eyes: **pass** : 0 / 0 (0%)

**Pull request closed without merging**